index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <link rel="stylesheet" href="style.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@700&display=swap" rel="stylesheet">
+</html>
+
+
+<body>
+    <div class="container">
+
+
+        
+        <div class="city1">
+            <img src="Huston.jfif" alt="Huston city" width="200px" height="200px">
```

Comment:
j.fif? what is mean?

Reply:
I've just edited it to .img ! Thank you for making me notice : )))